tools/benchmark.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# Performs benchmark, and append data to //gh-pages/data.json.
+
+import os
+import sys
+import json
+import time
+from util import run, run_output, root_path, build_path
+
+# The list of the tuples of the benchmark name and arguments
+benchmarks = [("hello", ["tests/002_hello.ts", "--reload"]),
+              ("relative_import", ["tests/003_relative_import.ts",
+                                   "--reload"])]
+
+data_file = "gh-pages/data.json"
```

Comment:
Maybe it's easier if you write this to `website/`, so that it's easy to view locally...

Add a comment to the top
```
# To view the benchmark locally run ./tools/http_server.py and visit 
# http://localhost:4545/website
```

Reply:
Changed to store data at `//website/data.json` and then export to `//gh-pages/`.

Now the local benchmark result should be visible at `http://localhost:4545/website` with `./tools/http_server.py` command!